repaygithub/ui-tools
# CACTUS-475 :: Dependency Upgrades

The eslint-config package will have a breaking change, but everything else should still work smoothly

### Testing

1. `cd modules/babel-preset && yalc publish`
2. `cd modules/repay-scripts && yalc publish`
3. `cd modules/eslint-config && yalc publish`
4. `cd modules/create-repay-ui && yalc publish`
5. `cd` into `cactus`
6. `yalc add @repay/babel-preset && yalc add @repay/scripts && yalc add @repay/eslint-config`
7. Run `yarn build` to make sure everything still builds properly
8. Run `yarn w mock-ebpp start` and/or `yarn w standard start` to make sure those apps still work just fine.
9. Run `yarn lint`.  You may need to change one of the eslint ignore comments, since it references a rule whose name has changed (`simple-import-sort/sort` -> `simple-import-sort/imports`).
10. `cd` into some random directory.  Run `create-repay-ui test-js-app`.  Select "JavaScript"
11. Run `yarn start` in the generated folder, and make sure everything works.
12. Repeat steps 10 and 11, but for a TypeScript app.
